### PR TITLE
add new txt file

### PR DESCRIPTION
**Pull request by AlexSuspect13**

### CODE REVIEW

**Review by mTeeeur — CHANGES_REQUESTED** [review]
why do we need to create an empty file?)

File: alxsuspect.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+misha zavali ebalo pls))) youre opinion donst
```
Comment (mTeeeur) [inline alxsuspect.txt:1]: cool

**Review by mTeeeur — APPROVED** [review]

File: alxsuspect.txt (hunk 1 above)
Comment (WhatWasThatt) [reply alxsuspect.txt:1]: why u use translit?

**Review by WhatWasThatt — CHANGES_REQUESTED** [review]

**Review by WhatWasThatt — APPROVED** [review]
zxc

**Pull request merged**